Source/UnitTests/Core/IOS/ES/FormatsTest.cpp

```diff
@@ -0,0 +1,227 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "Core/IOS/ES/Formats.h"
+#include "TestBinaryData.h"
```

Comment:
Does this work as a relative path like the one above it? (Ditto for the one in TestBinaryData.cpp)